src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.model.ValidationException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final String VALID_LOGIN = "Valid login";
+    private static final String VALID_PASSWORD = "Valid password";
+    private static final String INVALID_BORDERLINE_LOGIN = "user1";
+    private static final String INVALID_BORDERLINE_PASSWORD = "12345";
+    private static final int VALID_AGE = 18;
+    private static final int INVALID_BORDERLINE_AGE = 17;
+    private static final int INVALID_NEGATIVE_AGE = -2;
```

Comment:
You should avoid hardcoding values like the valid age. Instead, use a constant that represents the minimum legal age, as it might change over time.